### DISCUSSION TITLE
How do I get the ArchiveState?

### DISCUSSION THREAD

**Discussion by kasperbolarsen** [pnp/powershell#3983] [Q&A]
I found Set-PnPSiteArchiveState and it seems Get-SPOSite has ArchiveStatus . However, it doesn't look like Get-PnPSite has that ability yet, or am I missing something? (V2.4.0)

**Comment by gautamdsheth** [comment]
Get-PnPSite has includes parameter, maybe you can find the property there ?

**Reply by kasperbolarsen** [reply]
Not according to the link in the documentation , https://learn.microsoft.com/en-us/previous-versions/office/sharepoint-server/ee538579(v=office.15)

**Reply by gautamdsheth** [reply]
@kasperbolarsen  - should be available in tomorrow's nightly , you need to use `Get-PnPTenantSite -Url <site-url> -Detailed` and use `ArchiveStatus` property.

**Reply by spcodehelp** [reply]
@gautamdsheth - After updating to nightly build, I am getting below error message :
Get-PnPTenantSite: The property or field 'LoopOverrideSharingCapability' has not been initialized. It has not been requested or the request has not been executed. It may need to be explicitly requested.
Current Verion of PnP.PowerShell is 2.4.105.
Pls advise how would I get the "Archive Status" of SPO site using PnP PowerShell

**Reply by gautamdsheth** [reply]
@spcodehelp - the above approach will work. The error that you are facing has been fixed in the nightly builds available tomorrow.
